packages/auth/src/grant/service.ts

```diff
@@ -22,6 +22,7 @@ export interface GrantService {
     interactRef: string
   ): Promise<Grant | null>
   rejectGrant(grantId: string): Promise<Grant | null>
+  deleteGrant(continueId: string): Promise<number>
```

Comment:
```suggestion
  deleteGrant(continueId: string): Promise<boolean>
```
Since `number` is just an ORM thing, we should have the service throw or return a boolean depending on the result of the delete

Reply:
I had that at first but then decided to keep the number such that I can distinguish between the different scenarios on the route as well. Or would you do `true` for successfully deleted and `false` for not found?

Reply:
>Or would you do true for successfully deleted and false for not found?

:+1: